SU-HSDO/suhumsci
# SHS-4648: Fix image captions in events teasers covering event date

# READY FOR REVIEW

## Summary
- When captions are attached to event teaser they appear ontop of the event date. Event Date should always be ontop

## Need Review By (Date)
3/10

## Urgency
medium

## Steps to Test
1. The easiest way to test is to sync the EALC site `lando blt drupal:sync --site=ealc`
2. Go to `https://ealc.suhumsci.loc/events/east-asian-humanities-workshop-wu-mis-helen-troy-yiqun-zhou` and set the date as a future event.
3. Go to ![Screenshot from 2023-03-06 13-44-23](https://user-images.githubusercontent.com/369018/223217348-e65a8d94-fe3b-43c8-b704-90b9fcd8d62f.png)
`https://ealc.suhumsci.loc/` and verify the caption appears below the event date badge.

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
